cdc/puller/ddl_puller_test.go

```diff
@@ -660,15 +657,6 @@ func TestDDLPuller(t *testing.T) {
 	require.Equal(t, resolvedTs, uint64(30))
 	require.Nil(t, ddl)
 
-	inputDDL(t, ddlJobPullerImpl, &timodel.Job{
```

Comment:
Why we can remove this?

Reply:
In the new implementation, it is impossible to get the `create table` with status JobStateCancelled from tidb_ddl_history. Only the `create table` is done, the ddl can be inserted into tidb_ddl_history.